roles/overclicked/files/docker-compose.yaml

```
@@ -0,0 +1,14 @@
+services:
+  overclicked:
+#    image: ghcr.io/clicepfl/overclicked:latest
+    image: ghcr.io/parura5726/overclicked:main
+    environment:
+      # Change this only if sure that the new directory has the proper permissions
+      OVERCLICKED_DB_DIR: "/db"
+    volumes:
+      - "overclicked_db:/db"
+    ports:
+      - 8020:3000
+
+volumes:
+  overclicked_db:
```

The `overclicked_` prefix is unecessary; docker already adds it. (applying this suggestion also requires to update the volume name above).

```suggestion
  db:
```